### PR TITLE
OPICS Eval 6 (Ready for Review)

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Not planning to actually merge this. Just wanted to look at the code diffs.

### CODE REVIEW

File: mako/templates/ray_template_aws.yaml [hunk 1]
```diff
@@ -51,8 +51,9 @@ file_mounts: {
 
 setup_commands:
   - sudo cp /home/ubuntu/asound.conf /etc/asound.conf
-  - pip install --upgrade pip
-  - pip install -U "ray[default]"==1.13.0
+  - python3 -m pip install --upgrade pip
+  - python3 -m pip install -U "ray[default]"==1.13.0
+  - python3 -m pip install -U gpustat==1.0.0
```
Comment (ThomasSchellenbergNextCentury) [inline mako/templates/ray_template_aws.yaml:56]: I think these changes would be universally useful.

File: ray_scripts/pipeline_ray.py [hunk 2]
```diff
@@ -140,17 +140,18 @@ def run_scene(
...
+        history_files = f"{eval_dir}/SCENE_HISTORY/{scene_name}*.json"
+        scene_hist_matches = glob.glob(history_files)
 
         if len(scene_hist_matches) > 0:
             found_scene_hist = max(scene_hist_matches, key=os.path.getctime)
             hist_filename_no_ext = os.path.splitext(found_scene_hist)[0]
             timestamp = hist_filename_no_ext[-15:]
 
-            logging.info("History file timestamp: " + timestamp)
+            logging.info(f"History file found with timestamp: {timestamp}")
```
Comment (ThomasSchellenbergNextCentury) [inline ray_scripts/pipeline_ray.py:154]: Most of the changes in this file are to improve our log messages.

File: ray_scripts/pipeline_ray.py [hunk 3]
```diff
@@ -325,7 +325,7 @@ def __init__(self, args):
         self.exec_config = configparser.ConfigParser()
         self.exec_config.read(args.execution_config_file)
         self.disable_validation = args.disable_validation
-        self.dev_validation = args.dev_validation
+        self.dev_validation = False
```
Comment (ThomasSchellenbergNextCentury) [inline ray_scripts/pipeline_ray.py:328]: I'm assuming we'll address this in the `dev_validation` ticket.

File: run_eval.py [hunk 4]
```diff
@@ -497,7 +497,8 @@ def create_eval_set_from_folder(
         my_override = copy.deepcopy(override)
         scene_dir = os.path.join(base_dir, dir)
         if os.path.isdir(scene_dir):
-            my_override["log_name"] = f"{dir}-{metadata}.log"
+            prefix = f"{dir}-" if dir else ""
+            my_override["log_name"] = f"{prefix}{metadata}.log"
```
Comment (ThomasSchellenbergNextCentury) [inline run_eval.py:501]: If you're not using multiple clusters, this prevents the log file from being named `-level2.log` (it removes the dash).

**Comment by ThomasSchellenbergNextCentury** [comment]
This PR is now ready for review.

File: run_eval.py (hunk 4 above)
Comment (rartiss55) [reply run_eval.py:501]: For me the "-level2.log" config happened whenever I had a "/" at the end of my directory path in my run config yaml (so changing something like dirs: ["/home/ubuntu/charlie/"]  to ["/home/ubuntu/charlie"] fixed things for me).

That being said I'm not opposed to this change.

File: ray_scripts/pipeline_ray.py (hunk 3 above)
Comment (rartiss55) [reply ray_scripts/pipeline_ray.py:328]: Yeah I think this is resolved on the MESS PR + you can revert this bit.

File: configs/mcs_config_local_level2.ini [hunk 5]
```diff
@@ -2,8 +2,9 @@
 metadata=level2
 
 evaluation=false
-;s3_bucket=dev-evaluation-images
-;s3_folder=eval-35-test
+s3_bucket=dev-evaluation-images
```
[the remark this answers is not in the document]
Comment (rartiss55) [reply configs/mcs_config_local_level2.ini:5]: Thinking about this more, I made a ticket to get rid of the running locally instructions, I don't think they're as useful now (let me know if you disagree): https://nextcentury.atlassian.net/browse/MCS-1783.

**Comment by rartiss55** [comment]
I think besides reverting configs/mcs_config_local_level2.ini, LGTM!

**Review by jaudick — APPROVED** [review]

File: ray_scripts/pipeline_ray.py (hunk 3 above)
Comment (ThomasSchellenbergNextCentury) [reply ray_scripts/pipeline_ray.py]: Reverted.

File: configs/mcs_config_local_level2.ini (hunk 5 above)
[the remark this answers is not in the document]
Comment (ThomasSchellenbergNextCentury) [reply configs/mcs_config_local_level2.ini]: Just reverted these changes to avoid merge conflicts.

**Pull request merged**